tests/tests_lib.cairo

```diff
@@ -36,31 +36,51 @@ use openzeppelin::token::erc20::interface::{IERC20Dispatcher, IERC20DispatcherTr
 /// Mock Data
 ///
 
-fn get_mock_absorptions() -> Span<u128> {
-    let absorptions: Span<u128> = array![
-        0,
-        1000000000,
-        4799146600,
-        8882860500,
-        11843814000,
-        37092250700,
-        62340687400,
-        87589124100,
-        112837560800,
-        138085997600,
-        207617572100,
-        277149146600,
-        346680721200,
-        416212295700,
-        485743870300,
-        555275444800,
-        624807019300,
-        694338593900,
-        763870168400,
-        800000000000
+fn get_mock_absorptions() -> Span<u256> {
+    let absorptions: Span<u256> = array![
+        // 0,
+        // 1000000000,
+        // 4799146600,
+        // 8882860500,
+        // 11843814000,
+        // 37092250700,
+        // 62340687400,
+        // 87589124100,
+        // 112837560800,
+        // 138085997600,
+        // 207617572100,
+        // 277149146600,
+        // 346680721200,
+        // 416212295700,
+        // 485743870300,
+        // 555275444800,
+        // 624807019300,
+        // 694338593900,
+        // 763870168400,
+        // 800000000000
+        25000000000000000, // 25 000CC, in grams
```

Comment:
It might be useful to keep the old values for real-world tests as it looks like it was Manjarisoa absorptions.